run_sets/testsuite.yml

```diff
@@ -4,28 +4,28 @@
 # FIXME: ADD/REMOVE TO LIST STUFF
 --- # initialize suse-manager server
 - features/setup_user_and_proxy.feature
-- features/mainpage.feature
 # smdba before we start to sync channels
-- features/smdba.feature
 - features/channels_add.feature
 - features/push_package.feature
 - features/create_repository.feature
-- features/systemspage.feature
-- features/create_activationkey.feature
 - features/users.feature
 - features/credentials.feature
-
+- features/register_client.feature
+- features/minion_bootstrap.feature
+- features/docker_build_image.feature
 ## CORE feature end ###
 
 # SECONDARY FEATURE:
 
 # this feature can be runned in random order.
 # FIXME: MAKE THIS HAPPEN
 
-
-
+# server secondary 
+- features/mainpage.feature
+- features/create_activationkey.feature
```

Comment:
since this feature, create_activationkey create key that are reused, could be considered as core feature, imho is not because we can make feature create theirs act-key